src/ui/FuzzPanel.ts

```diff
@@ -597,13 +601,31 @@ export class FuzzPanel {
  *     ${"`"}yarn add nanofuzz/runtime -D${"`"}
  *  1. Add an import to the top of the file:
  *     ${"`"}import { FuzzTestResult } from "nanofuzz/runtime";${"`"}`;
+
+    const inArgs = fn.getArgDefs();
+    const validatorArgs = this.getValidatorArgs(inArgs);
+    const inArgConsts = inArgs
+      .map(
+        (argDef, i) =>
+          `  const ${argDef.getName()}: ${argDef.getTypeAnnotation()} = ${
+            validatorArgs.resultArgName
+          }.in[${i}];`
+      )
+      .join("\n");
+
+    const returnType = fn.getReturnType()?.type?.type;
+    const outArgConst = `  const out${returnType ? ": " + returnType : ""} = ${
```

Comment:
In the corner case where one of the function's arguments is named `out`, we could, e.g., de-conflict our (or the user's) use of `out` here, omit outputting this line (leaving it to the user to figure out), or outputting a comment saying the function's return value may be found in `out` of `validatorArgs.resultArgName`. What do you think might be best in this case?